Source/GUI/MemViewer/MemViewer.cpp

```diff
@@ -177,7 +227,7 @@ bool MemViewer::handleNaviguationKey(const int key)
     {
       scrollContentsBy(0, -1);
```

Comment:
same here, but you assign 15 this time, fixes the same thing, but down this time.